### DISCUSSION TITLE
Energia dostępna jako prosument od konkretnej daty

### DISCUSSION THREAD

**Discussion by grzywa91** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#115] [Q&A]
Cześć!
Czy jest możliwość w ramach dodanego skryptu obliczającego energię dostępną dla prosumenta dodać konkretną datę od kiedy ta energia ma być liczona? Chciałbym liczyć energię od ostatniego rozliczenia.

**Comment by PiotrMachowski** [comment]
Wystarczy użyć encji "konfigurowalnych", w nich można ustawić datę początkową
